hail/python/hail/methods/impex.py

```diff
@@ -2979,3 +2979,66 @@ def import_avro(paths, *, key=None, intervals=None):
         with DataFileReader(avro_file, DatumReader()) as data_file_reader:
             tr = ir.AvroTableReader(avro.schema.parse(data_file_reader.schema), paths, key, intervals)
     return Table(ir.TableRead(tr))
+
+
+
+@typecheck(spreadsheetID=oneof(str, sequenceof(str)), sheetname=str)
+def import_gsheet(spreadsheetID, sheetname) -> Table:
+    """Import sheet from a google sheets fle as a :class:`.Table` of strings.
+
+    Examples
+    --------
+
+    To import a google sheet as a table of strings:
+
+    >>> ht = hl.import_gsheet('1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms', 'Class Data')
+    >>> ht.describe()
+    ----------------------------------------
+    Global fields:
+        None
+    ----------------------------------------
+    Row fields:
+        'Student Name': str
+        'Gender': str
+        'Class Level': str
+        'Home State': str
+        'Major': str
+        'Extracurricular Activity': str
+    ----------------------------------------
+    Key: []
+    ----------------------------------------
+
+    Parameters
+    ----------
+    spreadsheetID: :class:`str` or :obj:`list` of :obj:`str`
```

Comment:
The spreadsheet id can only be a str, not a list of str, so let's change this to: 
```
spreadsheetID: :class:`str`
```